src/main/kotlin/blackjack/controller/Controller.kt

```diff
@@ -0,0 +1,116 @@
+package blackjack.controller
+
+import blackjack.model.Deck
+import blackjack.model.FinalResult
+import blackjack.model.GamblerInfo
+import blackjack.model.Player
+import blackjack.view.InputView
+import blackjack.view.OutputView
+
+class Controller() {
+    private var players: List<Player> = mutableListOf()
+    private val dealer: Player = Player(GamblerInfo("dealer"))
+    private val deck = Deck()
+
+    fun run() {
+        try {
+            createPlayers()
+            OutputView.displayNamesOfPlayers(players)
+            roundOne()
+            OutputView.displayCardsOfDealer(dealer)
+            players.forEach { OutputView.displayCardsOfPlayers(it) }
+            OutputView.printEmptyLine()
+            players.forEach { playerTakesTurn(it) }
+            dealerTakesTurn()
+            OutputView.printEmptyLine()
+            OutputView.displayCardsOfPlayersWithScore(dealer)
+            players.forEach { OutputView.displayCardsOfPlayersWithScore(it) }
+            OutputView.printEmptyLine()
+            OutputView.displayFinalResultsHeading()
+            showResults(FinalResult(dealer, players))
+        } catch (err: IllegalArgumentException) {
+            OutputView.displayErrorMessages(err.message)
+        }
+    }
+
+    private fun createPlayers() {
+        players =
+            processPlayerNames().map { Player(it) }
+    }
+
+    private fun roundOne() {
+        dealer.addCard(deck.drawCard(2))
+        players.forEach { it.addCard(deck.drawCard(2)) }
+    }
+
+    fun processPlayerNames(): List<GamblerInfo> {
+        repeat(MAX_ATTEMPTS) {
+            try {
+                val names = InputView.getNamesOfPlayers()
+                return names
+                    .split(",")
+                    .map(String::trim)
+                    .map { it -> GamblerInfo(it) } // to avoid abundant function: caller
```

Comment:
```suggestion
                    .map { GamblerInfo(it) } // to avoid abundant function: caller
```

It's redundant if you don't give the parameter a more meaningful name than `it`.

Reply:
Oh, thank you for catching it. I'll not use abundant `it`.